lib/modules/dosomething/dosomething_northstar/dosomething_northstar.module

```diff
@@ -104,6 +104,35 @@ function _dosomething_northstar_build_http_client() {
 /**
  * Send user profile updates to northstar.
  */
+// function dosomething_northstar_update_user($form_state) {
+//   $user = $form_state['user'];
+//   $id = $user->uid;
+
+//   $ns_user = dosomething_northstar_build_ns_user($user, $form_state);
+
+//   $northstar = new Northstar();
+//   $client = _dosomething_northstar_build_http_client();
+
+//   $northstar_user_info = dosomething_northstar_get_northstar_user($id);
+//   $northstar_user_info = json_decode($northstar_user_info->data, TRUE);
+//   $northstar_user_info = (object) $northstar_user_info['data'][0];
+//   $ns_id = $northstar_user_info->_id;
+
+//   $response = drupal_http_request($client['base_url'] . '/users/' . $ns_id, array(
+//     'headers' => $client['headers'],
+//     'method' => 'PUT',
+//     'data' => json_encode($ns_user),
+//     ));
+
+//   if ($response->code === '200' && module_exists('stathat')) {
+//     stathat_send_ez_count('drupal - Northstar - user updated - count', 1);
+//   }
+//   elseif ($response->code !== '200') {
+//     watchdog('dosomething_northstar', 'User not updated : ' . $response->code, NULL, WATCHDOG_ERROR);
```

Comment:
sorry - not finished working on this! I had a question above but it looks like it got hidden. Here it is below.

@angaither I think [this](https://github.com/DoSomething/northstar/blob/dev/app/Http/Controllers/UserController.php#L54)  might be preventing us from being able to use the /users POST endpoint. Would it be correct to add this to the block:
```
        if ($found_user && password_verify($input['password'], $found_user->password)) {
            $user = $found_user;
        } else {
        // check to make sure user is logged in
        }
```